src/v2/Apps/Artist/Routes/Works/index.tsx

```diff
@@ -77,9 +87,11 @@ export const WorksRouteFragmentContainer = createFragmentContainer(WorksRoute, {
         sizes: { type: "[ArtworkSizes]" }
         sort: { type: "String", defaultValue: "-partner_updated_at" }
         width: { type: "String" }
+        shouldFetchArtistSeriesData: { type: "Boolean!", defaultValue: false }
       ) {
       internalID
       ...ArtistTopWorksRail_artist
+      ...ArtistSeriesRail_artist @include(if: $shouldFetchArtistSeriesData)
```

Comment:
If the user doesn't have the lab feature, they don't need to fetch the fragment. Saves a little bit of data requesting and bandwidth. Did the same thing for the artwork page 'other works from this series' rail (https://github.com/artsy/force/blob/4301e22db2ffe9c5da0859c46e967621f65c69b4/src/v2/Apps/Artwork/routes.tsx#L17)